src/framework/theme/components/checkbox/checkbox-status.ts

```diff
@@ -0,0 +1,8 @@
+export enum NbCheckboxStatus {
```

Comment:
and this would be NbComponentStatus I believe? So that we can share it across the component.